src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj

```diff
@@ -79,9 +81,9 @@
         <Exec Command="npm run releaseBuild" />
     </Target>
     
-    <Target Name="Copy Documentation" AfterTargets="Build">
+    <!--<Target Name="Copy Documentation" AfterTargets="Build">
```

Comment:
Make sure to clean up this csproj file as well. Lines 51 and 63-66 look suspect to me.